### ISSUE TITLE
Idea: Bloomery

### ISSUE THREAD

**Issue by DisasterMoo** [TerraFirmaCraft/TerraFirmaCraft#77]
1.7.10 Bloomery, for me at least, was like: Hey, there's a gate for iron, but meh, that only looks like a gate, not that believable. So, I had an idea for 1.12+ TFC.

Why not use the following bloomery concept:

![](http://www.flintriflesmith.com/images/MiscArticleIllustrations/Anderson/pullingbloom4web.jpg)

Fired clay walls with circular metal to hold the structure. Also, a tuyere for wind intake(so the metal don't need to be damaged for each operation). The structure could be multiblock, with each added block adding more ore/fuel intake. 

Short version: A "tower" made of clay and worked bronze metal sheets to a circular shape. Items should be inputted at the top of the structure and slag outputs at the bottom.

**Comment by alcatrazEscapee** [comment]
At the risk of being blunt: You haven't actually presented any changes other than "this picture looks cool". The gate serves two main purposes: as the logic / controller block to house the TE, and as a progression gate from bronze -> iron.

Suggestions like this are better discussed on our [discord](https://invite.gg/terrafirmacraft).
